Userland/Applications/PDFViewer/main.cpp

```diff
@@ -6,25 +6,30 @@
  */
 
 #include "PDFViewerWidget.h"
+#include <LibCore/ArgsParser.h>
 #include <LibCore/System.h>
 #include <LibFileSystemAccessClient/Client.h>
 #include <LibGUI/Application.h>
 #include <LibGUI/Icon.h>
 #include <LibGUI/Menubar.h>
-#include <LibGUI/MessageBox.h>
 #include <LibGUI/Window.h>
 #include <LibMain/Main.h>
 
 ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
-    auto app = GUI::Application::construct(arguments);
+    TRY(Core::System::pledge("stdio recvfd sendfd rpath unix"));
+
+    auto app = TRY(GUI::Application::try_create(arguments));
     auto app_icon = GUI::Icon::default_icon("app-pdf-viewer");
 
     auto window = GUI::Window::construct();
     window->set_title("PDF Viewer");
     window->resize(640, 400);
 
-    TRY(Core::System::pledge("stdio recvfd sendfd rpath unix"));
+    const char* file_path = nullptr;
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(file_path, "PDF file to open", "path", Core::ArgsParser::Required::No);
+    args_parser.parse(arguments);
```

Comment:
move this to the top and keep the pledge where it was, that should "fix" autocomplete in shell until we figure out a way to either silence it or detect it ahead of time.